package.json

```diff
@@ -80,6 +80,7 @@
         "@babel/preset-env": "^7.12.11",
         "@babel/preset-typescript": "^7.12.7",
         "@casualbot/jest-sonar-reporter": "2.2.7",
+        "@jest/environment": "^29.7.0",
```

Comment:
What's this for? As far as I can see it's an empty package.

Reply:
its for the matrixRTC test environment: `matrix-js-sdk/spec/unit/matrixrtc/testEnvironment.ts`